client/src/components/NetworkDashboard.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+
+const NetworkDashboard = () => {
+    //Obtain Available Network Information from User's Browser
+    const networkInterface = navigator.connection 
+                             || navigator.mozConnection 
+                             || navigator.webkitConnection;
+
```

Comment:
I could use a bit more explanation around the 'navigator' where does it live? and those connections again? Maybe some multi line comments here would help?

Reply:
Its built into JS just like Object.method or Array.method. But under the hood, navigator requires access to the browser so its incompatible with node.

Reply:
ok yah i remember you referencing the simiarities to Object / Array. that makes sense. thanks!